client/landing/stepper/declarative-flow/sensei.ts

```diff
@@ -18,9 +18,9 @@ import { AssertConditionState, Flow } from './internals/types';
 import './internals/sensei.scss';
 
 function getStartUrl( step: string, locale: string ) {
-	return locale && locale !== 'en'
-		? `/start/account/user/${ locale }?redirect_to=/setup/${ SENSEI_FLOW }/${ step }`
-		: `/start/account/user?redirect_to=/setup/${ SENSEI_FLOW }/${ step }`;
+	const localeUrlPart = locale && locale !== 'en' ? `/${ locale }` : '';
+
+	return `/start/account/user/${ localeUrlPart }?redirect_to=/setup/${ SENSEI_FLOW }/${ step }&flow=${ SENSEI_FLOW }`;
```

Comment:
It looks like we'll have a double slash in this URL when there's a non-English locale.

Reply:
Thanks! I've updated it here https://github.com/Automattic/wp-calypso/pull/84191/commits/a536ddf58667200016a0b9f9ef765fd001449cf0.